<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-07-08T20:43:39 IST" name="Default suite" finished-at="2022-07-08T20:43:52 IST" duration-ms="12784">
    <groups>
    </groups>
    <test started-at="2022-07-08T20:43:39 IST" name="Default test" finished-at="2022-07-08T20:43:52 IST" duration-ms="12784">
      <class name="KiteLoginTest1">
        <test-method is-config="true" signature="openBrowser()[pri:0, instance:KiteLoginTest1@550dbc7a]" started-at="2022-07-08T20:43:39 IST" name="openBrowser" finished-at="2022-07-08T20:43:49 IST" duration-ms="9426" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method is-config="true" signature="logintoApp()[pri:0, instance:KiteLoginTest1@550dbc7a]" started-at="2022-07-08T20:43:49 IST" name="logintoApp" finished-at="2022-07-08T20:43:50 IST" duration-ms="1769" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logintoApp -->
        <test-method signature="verifyUserid()[pri:0, instance:KiteLoginTest1@550dbc7a]" started-at="2022-07-08T20:43:50 IST" name="verifyUserid" finished-at="2022-07-08T20:43:51 IST" duration-ms="136" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at library_files.UtilityClass.getTD(UtilityClass.java:24)
at KiteLoginTest1.verifyUserid(KiteLoginTest1.java:64)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserid -->
        <test-method is-config="true" signature="logoutFromApp(org.testng.ITestResult)[pri:0, instance:KiteLoginTest1@550dbc7a]" started-at="2022-07-08T20:43:51 IST" name="logoutFromApp" finished-at="2022-07-08T20:43:52 IST" duration-ms="1260" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyUserid status=FAILURE method=KiteLoginTest1.verifyUserid()[pri:0, instance:KiteLoginTest1@550dbc7a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromApp -->
        <test-method is-config="true" signature="logoutApp()[pri:0, instance:KiteLoginTest1@550dbc7a]" started-at="2022-07-08T20:43:52 IST" name="logoutApp" finished-at="2022-07-08T20:43:52 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutApp -->
      </class> <!-- KiteLoginTest1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
